python/ray/tune/suggest/suggestion.py

```diff
@@ -123,11 +128,14 @@ def set_search_properties(self, metric: Optional[str], mode: Optional[str],
         Args:
             metric (str): Metric to optimize
             mode (str): One of ["min", "max"]. Direction to optimize.
-            config (dict): Tune config dict.
+            config (dict): Tune config dict. This should ideally be used for
+            passing algorithm-specific parameters.
+            experiment (Optional[Experiment]): This is used for passing
+            algorithm-agnostic parameters. User should not need to touch this.
```

Comment:
The overflow lines need to be indented:
```suggestion
            config (dict): Tune config dict. This should ideally be used for
                passing algorithm-specific parameters.
            experiment (Optional[Experiment]): This is used for passing
                algorithm-agnostic parameters. User should not need to touch this.
```